doc/igmpproxy.conf.5.in

```diff
@@ -78,6 +78,15 @@ the risk of bandwidth saturation.
 .RE
 
 
+.B igmpv3
+.RS
+Enables IGMPv3 queries. On default settings, IGMPv1/2 queries are send out.
+With this setting, you can enable IGMPv3 frames for sending queries. This is needed
+for IGMPv3 capable hosts to send IGMPv3 reports. When a IGMPv3 capable host receives
+a IGMPv1/2 query, it answers with a IGMPv1/2 report.
+.RE
```

Comment:
This option affects downstream interfaces, so it should be in `phyint` section, not the global.